libs/community/langchain_community/chat_models/perplexity.py

```diff
@@ -63,10 +63,13 @@ class ChatPerplexity(BaseChatModel):
     """What sampling temperature to use."""
     model_kwargs: Dict[str, Any] = Field(default_factory=dict)
     """Holds any model parameters valid for `create` call not explicitly specified."""
-    pplx_api_key: Optional[str] = None
+    pplx_api_key: Optional[str] = Field(None, alias="api_key")
     """Base URL path for API requests, 
     leave blank if not using a proxy or service emulator."""
-    request_timeout: Optional[Union[float, Tuple[float, float]]] = None
+    request_timeout: Optional[Union[float, Tuple[float, float]]] = Field(
```

Comment:
Run `make format`  to prettify the content

Reply:
Thanks @eyurtsev just ran ruff on the file, and updated the pull request.